internal/agent/device/status/containers_test.go

```diff
@@ -126,14 +126,15 @@ var _ = Describe("containers exporter", func() {
 
 	BeforeEach(func() {
 		ctrl = gomock.NewController(GinkgoT())
-		deviceStatus = v1alpha1.DeviceStatus{}
+		deviceStatus = v1alpha1.DeviceStatus{Conditions: &[]v1alpha1.Condition{}}
 		execMock = executer.NewMockExecuter(ctrl)
 		container = newContainer(execMock)
 	})
 
 	Context("containers controller", func() {
 		It("list podman containers", func() {
```

Comment:
```suggestion
		It("should list podman containers", func() {
```

ginkgo idiomatic nit... I find it very nice when we use the describe/contect/it/should levels idiomatically, the errors are easier to understand.